quantum/keymap_extras/keymap_portuguese_osx_iso.h

```diff
@@ -127,11 +127,11 @@
 
 /* Alted symbols
  * ┌───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬───┬─────┐
- * │   │  │ @ │ € │ £ │ ‰ │ ¶ │ ÷ │ [ │ ] │ ≠ │ § │   │     │
+ * │   │  │ @ │ € │ £ │ ‰ │ ¶ │ ÷ │ [ │ ] │ ≠ │   │   │     │
```

Comment:
Sorry, this should be an NBSP. Either I didn't hit Option+Space or GitHub suggestions don't let you do it.

Reply:
Done! :)